coda-prototype/src/io/StorageUtils.js

```diff
@@ -0,0 +1,196 @@
+class StorageManager {
+    constructor() {
+        /*
```

Comment:
Why the large commented out code block?

Reply:
This file has yet to be refactored; these comments will be addressed in future when we improve storageUtils.